autonomy/cli/scaffold_fsm.py

```diff
@@ -1064,12 +1064,13 @@ class BehaviourTestsFileGenerator(AbstractFileGenerator):
             {rounds},
         )
 
-        from tests.conftest import ROOT_DIR
```

Comment:
decided to remove this entirely because it would be 
`ROOT_DIR = Path(__file__).parent.parent.parent.parent.parent`